src/invidious/channels/community.cr

```diff
@@ -24,7 +24,35 @@ def fetch_channel_community(ucid, cursor, locale, format, thin_mode)
   return extract_channel_community(items, ucid: ucid, locale: locale, format: format, thin_mode: thin_mode)
 end
 
-def extract_channel_community(items, *, ucid, locale, format, thin_mode)
+def fetch_channel_community_post(ucid, postId, locale, format, thin_mode, params : String | Nil = nil)
+  if params.nil?
+    object = {
+      "2:string"    => "community",
+      "25:embedded" => {
+        "22:string" => postId.to_s,
```

Comment:
Ditto my other review: please follow the Crystal convention of using `snake_case` for variable and function names.